packages/build-info/src/frameworks/observable.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseFramework, Category, Framework } from './framework.js'
+
+export class Observable extends BaseFramework implements Framework {
+  readonly id = 'observable'
+  name = 'Observable Framework'
+  npmDependencies = ['@observablehq/framework']
+  category = Category.SSG
+
+  dev = {
+    command: 'npm run dev',
```

Comment:
The commands should be here, something framework-specific like `next dev.`

Reply:
gotcha! updated that. I put `npm run dev` here because Solid, which I copied from, does the same: https://github.com/netlify/build/blob/cf751153a01cb2b9812ed9ca6b5b413849f1d842/packages/build-info/src/frameworks/solid.ts#L11